src/main/java/com/mitchej123/hodgepodge/mixins/late/hungeroverhaul/MixinHungerOverhaulModuleHarvestCraft.java

```diff
@@ -18,13 +18,13 @@
 @Mixin(ModuleHarvestCraft.class)
 public class MixinHungerOverhaulModuleHarvestCraft {
 
-    private static final Map<Item, FoodValues> foodValuesMap = new HashMap<Item, FoodValues>() {
```

Comment:
Not that I mind, but what's the objection to the anonymous inner class?

Reply:
double braces initializer is a famous antipattern. it creates a one time class that boost the memory usage, slow down class loading, confuse junior developers and make certain static analysis tool very unhappy.